src/config.js

```diff
@@ -256,7 +259,8 @@ export default class Config {
     this.modulesFolder = opts.modulesFolder;
     this.globalFolder = opts.globalFolder || constants.GLOBAL_MODULE_DIRECTORY;
     this.linkFolder = opts.linkFolder || constants.LINK_REGISTRY_DIRECTORY;
-    this.production = !!opts.production;
+    this.noProduction = opts.production === 'false' || opts.production === '0';
```

Comment:
@bestander Thanks for reviewing. I will make the change tonight to remove the `noProduction` variable as I realise it is not needed. I will also remove support for 0 and 1, and resolve conflicts.